### PR TITLE
[RFC] Put the generic configuration options in a separate page, to allow for shorter module specific man pages.

### PR DESCRIPTION
**Pull request by ericonr**

### CODE REVIEW

**Comment by Alexays** [comment]
This should also be done for the GitHub wiki 😃

File: man/waybar-sway-workspaces.5.scd [hunk 1]
```diff
@@ -65,10 +65,6 @@ Addressed by *sway/workspaces*
     default: empty ++
     Lists workspaces that should always be shown, even when non existent
 
-*on-update*: ++
```
Comment (Alexays) [inline man/waybar-sway-workspaces.5.scd:68]: Since sway/workspaces is not a Label module,  it has no common config, should we keep it here?

Comment (ericonr) [reply man/waybar-sway-workspaces.5.scd:68]: Updated to keep this config.

**Comment by ericonr** [comment]
Sorry for the delay! This is now updated with the new pages.

**Pull request closed without merging**